src/lib/flow/include/sol-flow-parser.h

```diff
@@ -107,16 +108,15 @@ int sol_flow_parser_del(
  * @brief Parses a flow description contained in @a buf and returns the resulting node type.
  *
  * @param parser The Flow Parser handle
- * @param buf Flow description to be parsed
- * @param len Buffer's size
+ * @param buf Flow description to be parsed. @note The buffer will be
```

Comment:
@note will create a note section in the middle of parameters definition. I found this weird. Is this desired? I prefer to add notes at the end of the comments.